src/main/java/dto/CreateBookRequestDto.java

```diff
@@ -1,12 +1,22 @@
 package dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
 
 public record CreateBookRequestDto(
+        @NotBlank
         String title,
+        @NotBlank
         String author,
+        @Pattern(regexp = "^\\d{3}-\\d{10}$", message = "ISBN must have the format 123-1234567890")
         String isbn,
+        @NotNull
+        @Positive
         BigDecimal price,
         String description,
         String coverImage
```

Comment:
pls add validation for max sizes here 

otherwise user input might cause error on DB layer